lib/practice_exercises.rb

```diff
@@ -1,13 +1,35 @@
-
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: The time complexity is O(n) because the algorithms time varies linearly with the length of the list array.
```

Comment:
Actually the time complexity is O(n<sup>2</sup>) because the `delete_at` method takes O(n) time.  This is because it has to shift all elements to the left one index.